feat: [Group 2] Update template pages

lyra-planet opened the pull request:
社区内容 2 组项目实战代码

kerthans committed:
feat(路由): 新增'/my/'路由配置

docs(组件): 添加TDesign组件TAvatar、TBadge、TIcon

refactor(布局): 重构layout/index.vue，添加底部导航栏和状态栏样式

feat(页面): 新增my/index.vue个人中心页面，包含用户信息和推荐服务

style(样式): 优化布局和响应式设计，适配安全区域

chore(资源): 添加Tdoc.svg、qq.svg、wechat.svg等图标资源

Mebius1916 committed:
feat: 基础UI搭建

Mebius1916 committed:
feat: 基础UI搭建完成

sym-666 committed:
feat: add first page and tab components

lyra-planet committed:
feat: just test

Mebius1916 committed:
feat: 交互优化

Mebius1916 committed:
feat: 交互优化

Mebius1916 committed:
feat: 用户信息持久化 && mock后端接口

sym-666 committed:
feat: mostly finish the second page and fix the first page

sym-666 committed:
feat: add the function to connect these pages

sym-666 committed:
fix: fix the error of the connection of the two pages

sym-666 committed:
feat: add a data.js,and all of data import this file to load data

Mebius1916 committed:
feat: 删除冗余页面

Mebius1916 committed:
feat: 用户信息持久化优化

Mebius1916 committed:
feat:  手机号采用E.164 标准格式

Mebius1916 committed:
feat:  接口问题修复

lyra-planet committed:
feat: test

kerthans committed:
feat(ui): 添加iOS风格状态栏图标并优化布局样式

添加电池、WiFi和信号强度SVG图标，替换原有模拟状态栏
实现实时时间显示功能
优化底部导航栏样式和交互效果
调整用户页面头像和菜单项的尺寸与颜色

kerthans committed:
feat(用户中心): 新增个人信息编辑和设置页面

添加个人信息编辑页面(/my/edit)和设置页面(/my/settings)的路由配置
实现个人信息编辑表单功能，包括头像上传、基础信息修改等
完成设置页面布局和功能模块划分
更新组件类型声明以支持新增功能
优化用户中心首页样式和交互逻辑

kerthans committed:
refactor(views): 优化用户信息页面样式和布局

- 重构标签显示逻辑，将星座和位置信息合并为标签
- 调整用户信息卡片的内边距和颜色样式
- 优化标签和统计卡片的布局和间距
- 改进服务项的文字显示和响应式布局

kerthans committed:
feat(my): 更新个人中心页面样式和功能

- 修改状态栏背景为透明并移除底部边界线
- 替换头部图标为小程序风格按钮并调整样式
- 更新地址选择器数据，增加全国省市选项
- 添加新的小程序图标资源文件
- 调整页面背景色和字体颜色

sym-666 committed:
fix: fix some bugs and improve the page structure

sym-666 committed:
feat: perfect the function of list and add the funcion of submitting.Necessary functions are realized

lyra-planet committed:
feat: finish

lyra-planet committed:
feat: finish

lyra-planet committed:
feat: finish

kerthans committed:
feat(编辑页): 重构个人信息编辑页面并添加地址数据

refactor(编辑页): 优化表单逻辑和日期处理
style(编辑页): 更新样式文件结构
docs(地址数据): 添加中国省市地址数据

lyra-planet committed:
Merge pull request #2 from lyra-planet/KaiCheng

Kai cheng

lyra-planet committed:
feat: finish

lyra-planet committed:
Merge pull request #3 from lyra-planet/KaiCheng

feat: finish

kerthans committed:
docs: 添加文档系统和个人中心模块文档

kerthans committed:
feat: 添加个人中心模块和文档网站

新增个人中心页面和设置页面，包含用户信息展示和编辑功能
添加VitePress文档网站，包含快速开始、布局系统等指南
更新路由配置和类型定义以支持新页面
添加SVG图标和地址数据组件
重构布局组件，添加iOS风格状态栏和底部导航
优化表单组件和样式，支持深色模式和响应式设计

kerthans committed:
build: 添加项目依赖项

kerthans committed:
build: 添加 vitepress 依赖

Mebius1916 committed:
Merge branch 'main' of github.com:lyra-planet/tdesign-mobile-vue-starter-community

Mebius1916 committed:
feat: 分支合并

Mebius1916 committed:
feat: 添加临时navbar

sym-666 committed:
feat: merge messagepage branch with chat functionality and status bar

kerthans committed:
refactor(router): 重构路由结构以使用布局组件

将路由结构重新组织以使用统一的布局组件，简化路由配置
移除冗余路由并合并登录相关页面到单一组件

sym-666 committed:
feat: 整合了代码

sym-666 committed:
Merge pull request #4 from lyra-planet/messagepage

Messagepage

Commit:
feat: 重构navbar布局，统一页面导航栏样式

- 将my页面的navbar抽离到layout中，实现统一管理
- 首页navbar中间显示搜索框，其他页面显示对应标题
- 发布页和对话页左侧显示返回按钮，其他页面显示view-list图标
- 对话页和设置页不显示底部导航栏
- 修复发布按钮跳转问题
- 优化navbar背景色和垂直居中对齐

Mebius1916 committed:
feat: 消息页数据mock

Mebius1916 committed:
feat: 分支合并

Mebius1916 committed:
Merge remote-tracking branch 'origin/main' into mebius

Mebius1916 committed:
feat: layout公共组件代码优化

Mebius1916 committed:
feat: mock-server代码结构优化

Mebius1916 committed:
feat: 消息相关api搭建

sym-666 committed:
feat: 添加了深色模式，适配了大部分界面

Mebius1916 committed:
feat: i18n插件应用及性能优化

lyra-planet committed:
feat: fix [home&publish]router&css

Mebius1916 committed:
feat: i18n插件应用及性能优化

Mebius1916 committed:
Merge branch 'main' of github.com:lyra-planet/tdesign-mobile-vue-starter-community

sym-666 committed:
feat: 完善了暗黑模式

sym-666 committed:
feat: 完善了暗黑模式

sym-666 committed:
feat:  merge

sym-666 committed:
feat: merge all change init main

sym-666 committed:
feat: finish merge

sym-666 committed:
feat: 组装深色模式到设置

sym-666 committed:
fix: 修复了重新进入界面的时候深色模式显示错误，更新了组件

sym-666 committed:
fix: 严格根据设计图重新实现了页面布局

Mebius1916 committed:
fix: layout代码优化

Mebius1916 committed:
Merge branch 'main' of github.com:lyra-planet/tdesign-mobile-vue-starter-community

kerthans committed:
fix(登录): 修改登录处理逻辑，未登录时跳转登录页

原逻辑在未登录时会自动模拟登录，现改为直接跳转登录页。已登录状态下跳转到个人信息编辑页

sym-666 committed:
feat: 创建了搜索页

sym-666 committed:
fix: 深色bug

kerthans committed:
feat(用户编辑): 重构用户信息编辑页面，使用TDesign表单组件

- 新增TForm组件类型声明
- 提取表单类型定义和hooks逻辑到独立文件
- 实现地址选择器和日期选择器组件
- 优化表单布局和样式
- 添加地区数据文件支持地址选择

lyra-planet committed:
feat: add menu

sym-666 committed:
feat: 完成了搜索页

sym-666 committed:
feat: 完成了搜索页

sym-666 committed:
feat: 完成了搜索页

Mebius1916 committed:
fix: 修复了一些交互上的问题

Mebius1916 committed:
feat: layout界面代码结构优化

Mebius1916 committed:
feat: 启动主应用同时启动mock服务

jin0209 commented on src/router/index.ts:
```diff
@@ -8,11 +9,99 @@ const router = createRouter({
+    // },
     {
       path: '/:pathMatch(.*)*',
       redirect: '/error/404',
```
访问未知路由会无限重定向。

jin0209 commented on src/router/index.ts:
```diff
@@ -1,5 +1,6 @@
 import { createRouter } from 'vue-router'
 import { handleHotUpdate, routes } from 'vue-router/auto-routes'
```
自动路由与手动路由并存且未使用
要么删除自动路由插件与引用；要么合并 ...routes 到当前路由表，减少维护成本

jin0209 commented on src/main.ts:
```diff
@@ -4,7 +4,8 @@ import { createApp } from 'vue'
 
 import { initGlobalConfig, injectStorageConfig } from '@/config'
 import * as directives from '@/directives'
```
src/directives/index.ts 导出的是 default {}，会迭代出 default 键并尝试注册为指令名
若无指令，删除自动注册逻辑；有指令则改为命名导出并正确遍历注册

jin0209 commented on src/api/request.ts:
```diff
@@ -0,0 +1,96 @@
+  private baseURL: string
+  private defaultHeaders: any
+
+  constructor(baseURL: string = 'http://localhost:3001/api') {
```
构建/部署环境切换不便，生产不可用。
建议：使用 import.meta.env（如 VITE_API_BASE）或 public/config.json 注入。

jin0209 commented on src/api/message.ts:
```diff
@@ -0,0 +1,27 @@
+
+// 获取未读消息总数
+export async function getUnreadCount(): Promise<ApiResponse<{ count: number }>> {
+  return get('/talklist/unread-count')
```
/talklist/unread-count mock中未实现？

Mebius1916 committed:
fix: 深色模式适配bug修复

Mebius1916 committed:
fix: 深色模式适配bug修复

Mebius1916 committed:
fix: 首页弹窗偏移量修复

Mebius1916 committed:
feat: components文件夹代码结构优化

Mebius1916 committed:
style: 删除冗余代码

Mebius1916 committed:
feat: 代码结构优化、与交互稿对其、业务功能优化等

1.优化my文件夹内代码结构。
2.优化了编辑页面地址的数据获取方式。
3.修复了与交互稿不同的交互bug。
4.对接用户登录鉴权以及登录后的信息展示。
5.添加编辑页面的表单提交接口并应用到mock服务。

Mebius1916 committed:
fix: edit页面使用layout并移除自定义navbar

jin0209 commented on src/api/request.ts:
```diff
@@ -0,0 +1,96 @@
+    delete this.defaultHeaders.Authorization
+  }
+
+  async request(endpoint: string, config: RequestConfig = {}) {
```
缺少超时/取消控制
建议引入 AbortController 和超时（如 3~8s），捕获并返回统一错误。

jin0209 commented on src/api/request.ts:
```diff
@@ -0,0 +1,96 @@
+        code: response.status,
+        message: data.message || (response.ok ? 'Success' : 'Error'),
+        data: data.data || data,
+        success: response.ok && data.code === 200,
```
success 同时依赖 response.ok 与 data.code === 200，与 mock 的 code 语义耦合，灵活性低
建议放宽成功条件：以 response.ok 为主，后端业务码通过单独字段判断或白名单映射。

jin0209 commented on src/api/request.ts:
```diff
@@ -0,0 +1,96 @@
+
+  // 设置认证token
+  setAuthToken(token: string) {
+    this.defaultHeaders.Authorization = `Bearer ${token}`
```
defaultHeaders 是实例级共享，若后续请求临时覆盖 headers，易互相污染
建议每次请求时合并 token，而不是长期修改默认头；或在 request 内动态注入，保证幂等

Commit:
feat: 添加数据中心页面功能

- 新增数据中心页面，包含整体情况、互动情况、完播率、按区域统计四个模块
- 配置数据中心路由
- 在'我的'页面添加数据中心服务跳转功能
- 配置数据中心页面标题和返回按钮
- 数据中心页面不显示底部导航栏

Mebius1916 committed:
Merge remote-tracking branch 'origin/Linmingjian' into mebius

Mebius1916 committed:
feat: 对其样式稿

Mebius1916 committed:
feat: 对其样式稿

Mebius1916 committed:
feat: 对其交互稿交互逻辑及其UI样式

Mebius1916 committed:
feat: i18n优化工作

Mebius1916 committed:
feat: mock数据提取 && api抽象

Mebius1916 committed:
refactor: 参考开源项目vue-admin-better对项目进行重构优化

Mebius1916 committed:
fix: i18n翻译优化

Mebius1916 committed:
fix: 消息页底部输入框及按钮对其样式稿

Mebius1916 committed:
fix: 消息页avatar对其样式稿

Mebius1916 committed:
style: 冗余代码删除

Mebius1916 committed:
style: 代码结构优化 && 删除冗余代码

Mebius1916 committed:
feat: 对接消息页api接口

Mebius1916 committed:
refactor: mock-server代码优化

Mebius1916 committed:
fix: loading样式优化

Mebius1916 committed:
feat: 消息页时间戳优化

Mebius1916 committed:
refactor: datacenter代码结构优化

Mebius1916 committed:
refactor: layout组件代码结构优化

Mebius1916 committed:
feat: 创建统一的消息封装并在生产环境去除console

Mebius1916 committed:
refactor: 将express重构为msw

Mebius1916 committed:
feat: 唤醒数字键盘优化

Mebius1916 committed:
refactor: 代码结构优化：删除mock-server与将public下的svg移动到assets

Mebius1916 committed:
feat: i18n缓存优化

Mebius1916 committed:
build: vite打包优化：

1.启用文本压缩：gzip与brotli。
2.启用image和svg压缩。
3.chunk拆包优化。

Mebius1916 committed:
perf: 首屏优化

1.异步加载地区库和区号库。
2.图片开启懒加载优化。

Mebius1916 committed:
perf: datacenter数据mock抽象api

Mebius1916 committed:
perf: 验证码登录页逻辑优化

Mebius1916 committed:
perf: 搜索页性能优化

1.防抖优化。
2.频繁请求导致的接口竞态问题优化。

Mebius1916 committed:
perf: 消息列表页虚拟滚动优化

1.编写公共虚拟滚动组件。
2.编写临时数据确认虚拟滚动应用成功。

Mebius1916 committed:
perf: 一些优化如下

1.手动拆包 dayjs、@vueuse；生产关闭 sourcemap。
2.移除未使用的 axios。
3.搜索页输入防抖 + AbortController 中止并发请求。
4.新增 useAppStorage；依据 NetworkConfig.storage 统一 local/session；Pinia 持久化跟随配置。

Mebius1916 committed:
feat: 新增httpCode页面

Mebius1916 committed:
perf: i18n持久化优化

Mebius1916 committed:
docs: 文档完成

Mebius1916 committed:
docs: 英文文档补全

Mebius1916 committed:
docs: 文档优化

Mebius1916 committed:
fix: 修复一个路由跳转bug

Mebius1916 committed:
perf: css按需引入

Mebius1916 committed:
polish: vitpress dark模式logo适配

Mebius1916 committed:
docs: 英文文档补全

jin0209 commented on src/config/index.ts:
```diff
@@ -1,52 +1,91 @@
 
-/** 存储 key 前缀 */
-const stroagePrefix = () => getGlobalConfig('storageNS')
+const storagePrefix = () => getGlobalConfig('storageNS')
```
拼写错误？ 应该是storagePrefix？

Mebius1916 replied:
导师您好这个已经在提交中更正了

Mebius1916 committed:
perf: 路由全部转为手动方便维护

jin0209 commented on src/api/auth.ts:
```diff
@@ -0,0 +1,32 @@
+}
+
+// 验证码登录
+export async function verifyCodeLogin(params: any) {
```
params类型应对齐 src/api/types.ts

Mebius1916 committed:
feat: 更新环境配置，移除网络配置文件并调整API基础路径

Mebius1916 committed:
feat: 添加超时取消功能

Mebius1916 committed:
feat: 增加认证token管理及业务成功码白名单功能

Mebius1916 committed:
fix: 环境变量问题修复

Mebius1916 committed:
perf: auth api的params对齐type.ts

Mebius1916 committed:
feat: 添加下拉刷新功能的多语言支持，并删除不再使用的SVG图标文件

Mebius1916 committed:
feat: 优化消息显示组件，调整偏移量和响应式布局，更新首页和搜索页样式

Mebius1916 committed:
feat: 更新多语言支持，优化底部导航组件样式，调整首页布局和响应式设计

Mebius1916 committed:
feat: 添加 vue-virtual-scroller 组件支持，优化虚拟列表的实现方式，更新相关属性和样式

Mebius1916 committed:
feat: 添加编辑页面及相关组件，优化地址选择器和表单处理逻辑，更新多语言支持和样式

Mebius1916 committed:
feat: 更新多语言支持，添加错误和空状态提示，优化组件中的文本处理逻辑

Mebius1916 committed:
feat: 更新虚拟列表组件，优化属性配置和事件处理，调整消息列表展示方式

Mebius1916 committed:
feat: 优化首页内容加载逻辑，添加虚拟列表支持，重构组件以提升性能和可维护性

Mebius1916 committed:
feat: 更新多语言支持，简化描述文本，优化发布组件样式

Mebius1916 committed:
fix: 调整message偏移量

Mebius1916 committed:
feat: 添加搜索历史清空功能，更新多语言支持，优化搜索页面样式

sym-666 committed:
feat: 补充了详细的项目文档

sym-666 committed:
feat: 补充了详细的项目文档

Mebius1916 committed:
feat: 更新文档，添加个人信息编辑和设置页面，优化列表三态处理，调整主题切换逻辑

Mebius1916 commented:
感谢老师的review，已参考评审建议优化项目。

Mebius1916 committed:
feat: 更新文档，调整目录结构和命名规范，优化错误处理和状态管理部分

Mebius1916 committed:
feat: 更新多语言支持，添加生产环境告警配置和空闲时间预取功能，优化代码分割策略

Mebius1916 committed:
feat: 添加lightningcss支持，更新vite配置以替代传统PostCSS流程，删除postcss.config.js文件，集成PWA功能以增强缓存管理

Mebius1916 committed:
feat: 更新index.html的viewport设置，启用TypeScript严格模式，优化handlers.ts中的用户信息处理，添加vue-virtual-scroller类型声明

Mebius1916 committed:
feat: 更新VirtualList组件，添加mode属性以支持动态和回收模式，优化文档说明和示例代码

Mebius1916 committed:
feat: 优化多语言加载逻辑，移除冗余缓存机制。

Mebius1916 committed:
feat: 添加请求取消功能，优化HttpClient类以支持请求ID管理，更新搜索功能以使用请求取消机制
